cluster/gce/util.sh

```diff
@@ -1303,7 +1303,17 @@ function test-teardown {
     --project "${PROJECT}" \
     --quiet \
     "${NODE_TAG}-${INSTANCE_PREFIX}-nodeports" || true
-  "${KUBE_ROOT}/cluster/kube-down.sh"
+  if [[ ${MULTIZONE:-} == "true" ]]; then
+      local zones=( $E2E_ZONES )
```

Comment:
@ixdy I need an array here rather than a toke list, so that I can iterate through it in reverse order.  Are you OK with that?